.github/workflows/CI.yml

```diff
@@ -117,6 +117,26 @@ jobs:
           command: fmt
           args: --all -- --check
 
+  clippy:
```

Comment:
graphene-sys fails because the ABI tests don't generate any layout tests... which seems to be problematic

```
error: unused import: `graphene_sys::*`
 --> graphene/sys/tests/abi.rs:8:5
  |
8 | use graphene_sys::*;
  |     ^^^^^^^^^^^^^^^
  |
  = note: `-D unused-imports` implied by `-D warnings`

error: unused imports: `align_of`, `size_of`
  --> graphene/sys/tests/abi.rs:11:16
   |
11 | use std::mem::{align_of, size_of};
   |                ^^^^^^^^  ^^^^^^^
```

Reply:
@GuillaumeGomez

Reply:
Not sure to get the problem here or how I can help...

Reply:
`graphene/sys/tests/abi.rs` contains any of the structs as it should, and you can get that problem if you try to build the tests :)

But I meant to CC @bilelmoussaoui, not you. Sorry